sonic_data_client/virtual_db.go

```diff
@@ -104,19 +153,34 @@ func v2rEthPortStats(paths []string) ([]tablePath, error) {
 	var tblPaths []tablePath
 	if strings.HasSuffix(paths[KeyIdx], "*") { // All Ethernet ports
 		for port, oid := range countersPortNameMap {
+			var oport string
+			if eport, ok := i2e_aliasMap[port]; ok {
+				oport = eport
+			} else {
+				log.V(2).Infof("%v does not have external alias", port)
+				oport = port
+			}
+
 			tblPath := tablePath{
-				dbName:       paths[DbIdx],
-				tableName:    paths[TblIdx],
-				tableKey:     oid,
-				delimitor:    separator,
-				jsonTableKey: port,
+				dbName:    paths[DbIdx],
+				tableName: paths[TblIdx],
+				tableKey:  oid,
+				delimitor: separator,
+				//jsonTableKey: port,
```

Comment:
do not leave comments, do not change existing code spacing unless necessary.

Reply:
I believe the changes in spacing are due to the use of "gofmt", which unified the code to the standards of the Go language. I believe we should keep these changes and continue using gofmt to ensure consistent style going forward.

Reply:
Agree. The changes are made due to the use of "gofmt", which targets to unify code formats.

Reply:
Is there a good reason to keep the commented-out line? If not, please remove.

Reply:
It was for me to track the changed place. Can remove it easily. Other comments?